gammapy/maps/region/ndmap.py

```diff
@@ -33,13 +34,15 @@ class RegionNDMap(Map):
     geom : `~gammapy.maps.RegionGeom`
         Region geometry object.
     data : `~numpy.ndarray`
-        Data array. If none then an empty array will be allocated.
+        Data array. If None then an empty array will be allocated.
```

Comment:
```suggestion
        Data array. If None, an empty array will be allocated.
```

Reply:
I decided to keep the 'then' format because there were lots of other docstrings written in this way